src/operator/tensor/sample_op.cc

```diff
@@ -31,52 +31,42 @@ MXNET_OPERATOR_REGISTER_SAMPLE(random_uniform, SampleUniformParam)
 .add_alias("uniform")
 .add_alias("_sample_uniform")
 .describe(R"code(Draw samples from a uniform distribution.
-
 Samples are uniformly distributed over the half-open interval [low, high)
-(includes low, but excludes high)::
-
-  nd.uniform(low=0, high=1, shape=(2,2)) = [[ 0.60276335,  0.85794562],
```

Comment:
- I thought it was simple enough that we didn't need an example for clarification.
- Also the code sample generates random samples, it is not reproducible by users and by unit tests
- Other sampling operators do not have examples.

If you feel strongly about having an example, we can add it back.